.github/workflows/fbgemm_nightly_build.yml

```diff
@@ -5,12 +5,14 @@ name: Push Binary Nightly
 
 on:
   # For debugging, enable push/pull_request
-  # [push, pull_request]
+  pull_request:
+    branches:
+      - main
   # run every day at 10:45 AM
-  schedule:
-    - cron:  '45 10 * * *'
+  # schedule:
```

Comment:
revert this testing changes?

Reply:
Thanks for your comment! It's still WIP. I will remove it once I can confirm this script change works on the ec2 server (currently failing)

Reply:
Removed since the test finished on the ec2 server reasonably.